.github/workflows/cicd-deploy-amplify-app.yaml

```diff
@@ -0,0 +1,137 @@
+name: "Deploy Amplify App"
+run-name: Deploy Amplify App ${{ inputs.deployEnvironment }} << ${{ github.ref_name }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      deployEnvironment:
+        description: Name of the environment to deploy
+        required: true
+        default: dev
+        type: string
+      terraformAction:
+        description: Terraform Action
+        required: true
+        default: apply
+        type: string
+      tag:
+        description: "This is the tag that is going to be deployed"
+        required: true
+        default: "latest"
+
+env:
+  PROJECT_NAME: notify-web-template-management
+  DEFAULT_ENVIRONMENT: dev
+  CI_PIPELINE_IID: ${{ github.run_number }}
+  CI_COMMIT_REF_NAME: ${{ github.ref_name }}
+  CI_COMMIT_BRANCH: ${{ github.ref_name }}
+  AWS_REGION: eu-west-2
+  DEPLOY_ENVIRONMENT: ${{ inputs.deployEnvironment }}
+  TERRAFORM_ACTION: ${{ inputs.terraformAction }}
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read  # This is required for actions/checkout
+
+concurrency:
+  group: notify-web-template-management-${{ github.workflow }}-${{ github.event.inputs.deployEnvironment }}
+  cancel-in-progress: false
+
+jobs:
+  metadata:
```

Comment:
Is it worth putting `metadata` and `list_variables` into an action so it can be reused across the two amplify workflows?

Reply:
not changing as discussed - moving this into an action would create more long lists of variables than it removes and make the code harder to maintain